specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/ContainerApps.json

```diff
@@ -871,6 +871,23 @@
             "enableMetrics": {
               "description": "Enable jmx core metrics for the java app",
               "type": "boolean"
+            },
+            "javaAgent": {
+              "description": "Diagnostic capabilities achieved by java agent",
+              "type": "object",
+              "properties": {
+                "enabled": {
+                  "description": "Enable java agent injection for the java app.",
```

Comment:
>": [](http://example.com/codeflow?start=30&length=2)

[https://github.com/Azure/azure-openapi-validator/blob/main/docs/enum-instead-of-boolean.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/enum-instead-of-boolean.md)

Reply:
thx for your feedback. It's intended to keep consistent with the `enableMetrics` property which is also boolean.